src/browser-installer/chrome/browser.ts

```diff
@@ -0,0 +1,61 @@
+import { resolveBuildId, canDownload, install as puppeteerInstall } from "@puppeteer/browsers";
+import { MIN_CHROME_FOR_TESTING_VERSION } from "../constants";
```

Comment:
No.
We have these chrome versions:
- MIN_CHROMIUM_VERSION (73)
- MIN_CHROMIUM_MAC_ARM_VERSION (93)
- MIN_CHROME_FOR_TESTING_VERSION (113)

- [73, 92]. Minimal chrome to use. Chromium is used. (mac x64 version for mac arm, because there is no mac arm for these versions)
- [93, 112]. Chromium is still used, but now Mac_arm version for mac.
- 113+. Chrome for testing is used.